src/scse/default_run_parameters/national_grid_default_run_parameters.py

```diff
@@ -16,6 +16,17 @@ class _RunParameters(CoreRunParameters):
     simulation_logging_level = 'CRITICAL'
     num_batteries = 1  # 5
 
+    # Penalty and reward prices, w/ units £/MWh
+    source_request_reward_penalty = -35.63
+    sink_deposit_reward_penalty = 27.63
+    battery_drawdown_reward_penalty = 35.63
+    battery_charging_reward_penalty = -27.63
```

Comment:
@collinskatie The positive value I'd originally put in my diagram for 2021 should have been `£115.75`. However, 2021 was pretty wild - the high positive value was the result of some enormous outliers. The 2020 values were more stable.